kuma/static/js/contribution-faq.js

```diff
@@ -36,20 +36,21 @@
         $(event.target).prev().removeClass('active');
         $(event.target).addClass('active');
 
-        sendAnalyticsVoteEvent('question_' + questionNumber, null, 0);
+        sendAnalyticsVoteEvent('FAQ vote', 'question_' + questionNumber, 0);
     }
 
 
     function onFeedback(event) {
         event.preventDefault();
 
-        var feedback = $(this).find('textarea').val() || '';
-        var action = $(this).find('textarea').attr('data-action') || '';
-        mdn.analytics.trackEvent({
-            category: 'Contribution feedback',
-            action: action,
-            label: feedback,
-        });
+        var feedback = $(this).find('textarea').val();
```

Comment:
We know the `textarea` has an id of `faq3` so, it would be more performant to do:

```
var feedback = document.getElementById('faq3').value;
```